app/components/newsletter/newsletter-header-overview.hbs

```diff
@@ -1,37 +1,35 @@
-{{!-- TODO: octane-refactor --}}
-{{!-- template-lint-disable no-action --}}
 <Auk::Navbar @skin="gray-100" @auto={{true}}>
   <Auk::Toolbar>
     <Auk::Toolbar::Group @position="left">
       <Auk::Toolbar::Item>
         <div class="auk-o-flex auk-o-flex--vertical auk-u-mt-2">
           <div class="auk-o-flex auk-o-flex--vertical-center">
-            <h4 class="auk-toolbar-complex__title auk-u-text-capitalize" data-test-newsletter-header-overview-title>
+            <h4
+              class="auk-toolbar-complex__title auk-u-text-capitalize"
+              data-test-newsletter-header-overview-title
+            >
               {{t "newsletter-overview-pdf-name"}}
               -
-              {{await this.agenda.createdFor.kindToShow.label}}
-              {{moment-format this.agenda.createdFor.plannedStart "DD MMMM YYYY"}}
+              {{await @agenda.createdFor.kindToShow.label}}
+              {{moment-format @agenda.createdFor.plannedStart "DD MMMM YYYY"}}
```

Comment:
Buiten scope van de octane refactor, maar elk voorval van `agenda.createdFor` == `meeting`.
We hebben die waarde, maar we gebruiken die niet? Mogelijk is die meeting er pas later bijgekomen en hebben we de bestaande code toen niet gerefactored.